desc "run tests"
lane :executetest do
begin
            # Something where you'd like to be able to react to failures

   scan(
    scheme: "iOSProjectTests",
    devices: ["iPhone X"],
    output_style: "standard")
            # If ANY exception happens, let me handle it
            rescue => ex
            # re-raise the exception unless it's something we're OK with ignoring
            # raise ex unless ex.message.include?('Error message of interest')
            # UI.error ex
            # UI.user_error!("Lint check or Test case failed, but continuing anyway!")
        UI.important("There were a few failing test cases. Continuing execution")

        end
end


